### PR TITLE
Reduces API paging size from 250 to 100

### PR DESCRIPTION
**Pull request by jcbalmeida**

### CODE REVIEW

File: temba/settings.py.prod [hunk 1]
```diff
@@ -77,6 +77,7 @@ REST_FRAMEWORK["DEFAULT_THROTTLE_RATES"]["v2.messages"] = env("V2_MESSAGES", def
 REST_FRAMEWORK["DEFAULT_THROTTLE_RATES"]["v2.broadcasts"] = env("V2_BROADCASTS", default="36000/hour")
 REST_FRAMEWORK["DEFAULT_THROTTLE_RATES"]["v2.runs"] = env("V2_RUNS", default="3000/hour")
 REST_FRAMEWORK["DEFAULT_THROTTLE_RATES"]["v2.api"] = env("V2_API", default="3000/hour")
+REST_FRAMEWORK["PAGE_SIZE"] = 100
```
Comment (matmsa27) [inline temba/settings.py.prod:80]: pls change the integer defined to an environment variable, with a value default with 250, and on configmap add the number 100

Comment (paulobernardoaf) [reply temba/settings.py.prod:80]: the default value shouldn't be 100 for the environment variable @matmsa27 ? since this change is to reduce this value

Comment (jcbalmeida) [reply temba/settings.py.prod]: Since we are testing this change performance impact, i think keeping 250 as default is the better option.

**Review by paulobernardoaf — APPROVED** [review]

File: temba/settings.py.prod (hunk 1 above)
Comment (matmsa27) [reply temba/settings.py.prod]: let's keep 250 with value default, we can treat this on the application environment.

File: temba/settings.py.prod [hunk 2]
```diff
@@ -77,6 +77,7 @@ REST_FRAMEWORK["DEFAULT_THROTTLE_RATES"]["v2.messages"] = env("V2_MESSAGES", def
 REST_FRAMEWORK["DEFAULT_THROTTLE_RATES"]["v2.broadcasts"] = env("V2_BROADCASTS", default="36000/hour")
 REST_FRAMEWORK["DEFAULT_THROTTLE_RATES"]["v2.runs"] = env("V2_RUNS", default="3000/hour")
 REST_FRAMEWORK["DEFAULT_THROTTLE_RATES"]["v2.api"] = env("V2_API", default="3000/hour")
+REST_FRAMEWORK["PAGE_SIZE"] = env.int('REST_FRAMEWORK_PAGE_SIZE', default=250)
```
Comment (matmsa27) [inline temba/settings.py.prod:80]: is it necessary to use `env.int`?

Comment (jcbalmeida) [reply temba/settings.py.prod:80]: by default, env variables are parsed to str. It's the same as env.bool()

**Pull request merged**